packages/ai-ide/src/common/architect-prompt-template.ts

```diff
@@ -44,3 +46,26 @@ Always look at the relevant files to understand your task using the function ~{$
 {{prompt:project-info}}
 `
 };
+
+export const architectTaskSummaryPromptTemplate: PromptTemplate = {
+    id: ARCHITECT_TASK_SUMMARY_PROMPT_TEMPLATE_ID,
+    template: `{{!-- This prompt is licensed under the MIT License (https://opensource.org/license/mit).
+Made improvements or adaptations to this prompt template? We'd love for you to share it with the community! Contribute back here:
+https://github.com/eclipse-theia/theia/discussions/new?category=prompt-template-contribution --}}
+
+Your task is to analyze the current chat session and summarize it to prepare completing the coding task.
```

Comment:
```suggestion
Your task is to analyze the current chat session and summarize it to prepare to complete the coding task.
```

Or maybe `to prepare for the next phase of` or `next step of`?